src/datanode/src/alive_keeper.rs

```diff
@@ -354,46 +357,38 @@ struct CountdownTask {
 }
 
 impl CountdownTask {
-    // returns true if region closed successfully
-    async fn run(&mut self) -> Option<bool> {
+    async fn run(&mut self) {
         // 30 years. See `Instant::far_future`.
         let far_future = Instant::now() + Duration::from_secs(86400 * 365 * 30);
 
         // Make sure the alive countdown is not gonna happen before heartbeat task is started (the
         // "start countdown" command will be sent from heartbeat task).
         let countdown = tokio::time::sleep_until(far_future);
         tokio::pin!(countdown);
-
         let region_id = self.region_id;
+
+        let mut started = false;
```

Comment:
What's the use of this `started`?

Reply:
The start method can be called multiple times.